felles/src/main/kotlin/no/nav/familie/kontrakter/felles/oppgave/FinnOppgaveRequest.kt

```diff
@@ -1,18 +1,29 @@
 package no.nav.familie.kontrakter.felles.oppgave
 
-class FinnOppgaveRequest(val tema: String? = null,
-                         val behandlingstema: String? = null,
-                         val oppgavetype: String? = null,
-                         val enhet: String? = null,
-                         val saksbehandler: String? = null,
-                         val journalpostId: String? = null,
-                         val tilordnetRessurs: String? = null,
-                         val tildeltRessurs: Boolean? = null,
-                         val opprettetFomTidspunkt: String? = null,
-                         val opprettetTomTidspunkt: String? = null,
-                         val fristFomDato: String? = null,
-                         val fristTomDato: String? = null,
-                         val aktivFomDato: String? = null,
-                         val aktivTomDato: String? = null,
-                         val limit: Long? = null,
-                         val offset: Long? = null)
+import no.nav.familie.kontrakter.felles.abstraction.QueryObject
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class FinnOppgaveRequest(val tema: Tema,
+                              val behandlingstema: Behandlingstema? = null,
+                              val oppgavetype: Oppgavetype? = null,
+                              val enhet: String? = null,
```

Comment:
Hva er grunnen at disse skal gjøres om til query params? For å ikke trenge å mappe om de til query params i OppgaveRestClient? 

Det virker som att noen av disse mappes om i integrasjoner, OppgaveRestClient
* enhet -> tildeltEnhetsnr
* saksbehandler -> tilordnetRessurs
* opprettet*Tidspunkt -> opprettet*
* fristFomDato, fristTomDato, aktivFomDato, aktivTomDato

Reply:
Fordi de skal brukes i en getReequest. Dagens metode er laget med Post for å bruke en dto, med et queryParams oppnår man en korrekt getRequest, i tillegg til at dagens metode bruker Strign til alle variabler slik at validering av input må gjøres manuelt.